src/styles/blocks/contacts.scss

```diff
@@ -1,4 +1,9 @@
 .contacts {
+  &__form {
+    margin: 0 auto;
+    max-width: 600px;
```

Comment:
it is `411px`

Reply:
but what do you mean, it looks ok on desktop
![image](https://user-images.githubusercontent.com/44649686/209987231-d23b8597-79f1-42e0-9a73-0c73c1b7510b.png)